### ISSUE TITLE
Bug: Register form not completely visible

### ISSUE THREAD

**Issue by Rahulm2310** [anitab-org/stem-diverse-tv-cms#30]
### Describe the bug

The register UI is not completely visible on laptop screen size and not scrollable. Also, the alerts are not completely visible and just shows up for merely 1 second. 

### To Reproduce
1. Go to https://anitab-org.github.io/stem-diverse-tv-cms/
2. Click on 'register'

### Expected behavior

The page should be scrollable and alerts should remain for more time like 5 seconds or more.

### Screenshots

![register-ui-issue](https://user-images.githubusercontent.com/54268438/107839071-74ca3980-6dcf-11eb-8ec9-d75807802186.gif)

### Desktop (please complete the following information):
 - OS: Windows 10
 - Browser : Chrome
 - Version: 84

**Comment by Rahulm2310** [comment]
@annabauza Can you please confirm/approve this issue. This can be a good first timer issue.

**Comment by chinmaym07** [comment]
@Rahulm2310 have you tested these locally .?
Also the error that displays for a second that's not a error that's a confirmation that password matched & It should go like that , it occurs on the hosted because we are not getting back the correct response from the backend. Try this locally you will get it .
The errors are shown in red & confirmation is shown in blue .

![Peek 2021-02-13 14-36](https://user-images.githubusercontent.com/44670961/107846331-ed99b780-6e08-11eb-9cda-acfa1e16e838.gif)
Register UI is completely visible .Can you please exactly point where the error is.?

**Comment by Rahulm2310** [comment]
@chinmaym07 On my 14-inches screen, I face the issue and have added the GIF for the same. May be you possess a larger display. 

>Also the error that displays for a second that's not a error that's a confirmation that password matched

I mentioned alert and not error. I think the message even if a confirmation should stay at least for 2 seconds. Otherwise there is no point displaying them.

**Comment by chinmaym07** [comment]
@Rahulm2310 It just a sign of notifying the user that he had entered correct credentials & in last GIF that I posted it stays there for more than 2-3 seconds & the GIF you posted above looks blurred to me sorry I am unable to find where the issue is with displaying of Registeration form. 
Please check it once more , If you still face this issue of bad visibility of Registeration form.

**Comment by annabauza** [comment]
approved. I don't think it's first timer issue.

**Comment by jalajcodes** [comment]
@annabauza Can I work on it?

**Comment by annabauza** [comment]
@jalajcodes yours

**Comment by jalajcodes** [comment]
@Rahulm2310 Do you agree with what @chinmaym07 said about the errors and confirmation?

In your case, the error stays for just a fraction because the request never gets sent to the correct url.  This should be fixed soon.

**Comment by jalajcodes** [comment]
I think its not necessary to show a confirmation when passwords match... instead, it should check if they don't match and then show the error and return from the function so the request doesn't get sent.

**Comment by Rahulm2310** [comment]
@jalajcodes Are you working on this. If not started, you can make it available for OSH.

**Comment by jalajcodes** [comment]
@Rahulm2310 yeah that's what I want to do, just waiting to get access to label issue😉

**Comment by Rahulm2310** [comment]
@jalajcodes unassigning you and opening this for OSH aspirants 😉

**Comment by nlok5923** [comment]
@Rahulm2310 can i take up this issue.
